pkg/apis/performanceprofile/v2/performanceprofile_validation.go

```diff
@@ -323,5 +324,11 @@ func (r *PerformanceProfile) validateWorkloadHints() field.ErrorList {
 		}
 	}
 
+	if r.Spec.WorkloadHints.HighPowerConsumption != nil && *r.Spec.WorkloadHints.HighPowerConsumption {
+		if r.Spec.WorkloadHints.PerPodPowerManagement != nil && *r.Spec.WorkloadHints.PerPodPowerManagement {
+			allErrs = append(allErrs, field.Invalid(field.NewPath("spec.workloadHints.HighPowerConsumption"), r.Spec.WorkloadHints.HighPowerConsumption, "Invalid WorkloadHints configuration: HighPowerConsumption to enabled and PerPodPowerManagement enabled"))
+		}
```

Comment:
suggestion for the error message:  HighPowerConsumption and PerPodPowerManagement can not be both enabled